### ISSUE TITLE
react-redux 7 uses useMemo, all Connected components complaining

### ISSUE THREAD

**Issue by joelstransky** [welldone-software/why-did-you-render#18]
Great job on the new release!
I just updated react-redux to 7.0.1 and WDYR is complaining about every single one. How should we handle this?
```
Connect(Header)
whyDidYouRender.min.js:8 {Connect(Header): ƒ} "Re-rendered because of hook changes:"
whyDidYouRender.min.js:8 hook useMemo 
whyDidYouRender.min.js:8 different React elements with the same displayName. (more info at http://bit.ly/wdyr3)
whyDidYouRender.min.js:8 {"prev ": {…}} "!==" {"next ": {…}}
```

**Comment by vzaidman** [comment]
I'll update my projects to react-redux very soon as well so i'll fix this one shortly

**Comment by joelstransky** [comment]
For now I'm just filtering it out.
```
if (__DEV__) {
  const whyDidYouRender = require('@welldone-software/why-did-you-render')
  whyDidYouRender(React, { include: [/^.*$/], exclude: [/^Switch/, /^Router/, /^Route/, /^Connect/] })
}
```

**Comment by vzaidman** [comment]
I believe we discovered a small performance issue with react-redux :)
 I fixed it there.

**Comment by vzaidman** [comment]
solved in react-redux 7.0.2 (by me :P)

**Comment by vzaidman** [comment]
also notice this article [Track React-Redux Redundant Re-Renders With “Why Did You Render” Version 3](http://bit.ly/wdyr04)
